src/stories/vitalCard.stories.js

```diff
@@ -0,0 +1,48 @@
+import { html } from 'lit';
+import '@kyndryl-design-system/shidoka-foundation/css/grid.css';
+import '@kyndryl-design-system/shidoka-foundation/components/card';
+
+import './sampleCardComponents/vitalCard.sample.ts';
+//import './sampleCardComponents/vitalCard.mobile.sample.ts';
+
+export default {
+  title: 'Patterns/Vital Card',
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/CQuDZEeLiuGiALvCWjAKlu/Applications---Component-Library?type=design&node-id=9941-18758&mode=design&t=N5Lw7kAKTAjoCa0x-0',
+    },
+  },
+};
+
+export const Default = {
+  render: () => {
+    return html`<kd-card type="normal" href="" target="" rel="">
+      <vital-card-sample-component></vital-card-sample-component>
+    </kd-card>`;
+  },
+};
+
+export const Clickable = {
+  render: () => {
+    return html`<kd-card type="clickable" href="" target="" rel="">
+      <vital-card-sample-component></vital-card-sample-component
+    ></kd-card>`;
+  },
+};
```

Comment:
I think we can actually remove the clickable example. It doesn't really offer anything that can't already be found in the Card docs.

Reply:
Yeah having two stories showing the same pattern is not necessary IMO.

Reply:
Done